src/main/java/com/jmb/springfactory/service/comment/CommentServiceImpl.java

```diff
@@ -3,21 +3,29 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
 import com.jmb.springfactory.dao.GenericMySQLService;
 import com.jmb.springfactory.dao.comment.CommentMySQLService;
+import com.jmb.springfactory.dao.group.GroupMongoService;
 import com.jmb.springfactory.dao.task.TaskMySQLService;
+import com.jmb.springfactory.dao.user.UserMongoService;
 import com.jmb.springfactory.exceptions.NotFoundException;
 import com.jmb.springfactory.exceptions.PersistenceLayerException;
 import com.jmb.springfactory.exceptions.ServiceLayerException;
 import com.jmb.springfactory.model.bo.BusinessObjectBase;
 import com.jmb.springfactory.model.dto.CommentDto;
+import com.jmb.springfactory.model.dto.UserDto;
+import com.jmb.springfactory.model.dto.WorkGroupDto;
+import com.jmb.springfactory.model.dto.WorkLogDto;
 import com.jmb.springfactory.model.entity.Comment;
 import com.jmb.springfactory.model.entity.Task;
+import com.jmb.springfactory.model.entity.User;
+import com.jmb.springfactory.model.entity.WorkGroup;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.jmb.springfactory.model.entity.WorkGroup'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)